tsconfig.json

```diff
@@ -24,6 +24,7 @@
     "noImplicitThis": true,
     "noImplicitAny": true,
     "strictNullChecks": true,
+    "esModuleInterop": true
```

Comment:
This is key to make the test pass, without it it shows the following error: `TypeError: Cannot read property 'createElement' of undefined`

Reply:
Nice find!